src/app/games/dvonn/tests/DvonnPieceStack.spec.ts

```diff
@@ -54,12 +54,6 @@ describe('DvonnPieceStack', () => {
             expect(stack2.getOwner()).toBe(Player.ONE);
         });
     });
-    describe('belongsTo', () => {
```

Comment:
why the removal ?

Reply:
c'était un doublon, l'autre test test de belongsTo étant strictement meilleur !